### PR TITLE
Adding icons for openplantbook

### PR DESCRIPTION
**Pull request by Olen**
## Proposed change

Add icons for the openplantbook integration

## Type of change

- [X] Add a new logo or icon for a new core integration
- [ ] Add a missing icon or logo for an existing core integration
- [ ] Add a new logo or icon for a custom integration (custom component)
  - [ ] I've opened up a PR for my custom integration on the [Home Assistant
    Python wheels repository](https://github.com/home-assistant/wheels-custom-integrations)
- [ ] Replace an existing icon or logo with a higher quality version
- [ ] Removing an icon or logo

## Additional information

- This PR fixes or closes issue: fixes #
- Link to code base pull request: https://github.com/home-assistant/core/pull/42359
- Link to documentation pull request: https://github.com/home-assistant/home-assistant.io/pull/15445
- Link to integration documentation on our website: 

## Checklist

- [X] The added/replaced image(s) are **PNG**
- [X] Icon image size is 256x256px (`icon.png`)
- [X] hDPI icon image size is 512x512px for  (`icon@2x.png`)
- [ ] Logo image size has min 128px, but max 256px, on the shortest side (`logo.png`)
- [ ] hDPI logo image size has min 256px, but max 512px, on the shortest side (`logo@2x.png`)

### CODE REVIEW

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by Olen** [comment]
Sorry for dismissing review.  The upstream project sent some new logos and icons to use.

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**
